examples/07_load_finetune.yaml

```diff
@@ -68,3 +68,20 @@ exp2-finetune-model: !LoadSerialized
   - path: train.dev_zero
     val: True
 
+exp3-finetune-minrisk: !LoadSerialized
```

Comment:
This should probably go into a new, dedicated minrisk example config.

Reply:
Agree

Reply:
done